### PR TITLE
Nelson - Adds templates for github issues

### PR DESCRIPTION
**Pull request by diablouma**
Adds templates for github issues for: user stories, bugs.
Adds template for PRs

### CODE REVIEW

File: dashboard/apps/web/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const webpack = require('webpack')
```
Comment (sauljabin) [inline dashboard/apps/web/server.js:1]: 😆 Guess this is not realated

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+**Context:**
+Describe the context and any relevant information about this user story
+
+**Acceptance Criteria:**
+**Given** [how things begin]
+**When** [action taken]
+**Then** [outcome]
+
+**Scope**
```
Comment (sauljabin) [inline .github/ISSUE_TEMPLATE/user_story.md:22]: let's add a section that says what is the component:
- Server
- Dashboard
- etc

**Review by sauljabin — APPROVED** [review]

File: dashboard/apps/web/server.js (hunk 1 above)
Comment (diablouma) [reply dashboard/apps/web/server.js]: buen catch, thanks Saul

File: .github/ISSUE_TEMPLATE/user_story.md (hunk 2 above)
Comment (diablouma) [reply .github/ISSUE_TEMPLATE/user_story.md:22]: done

**Pull request merged**